spec/credit_card_validations_spec.rb

```diff
@@ -105,53 +105,62 @@
 
   describe "adding/removing brand" do
 
+    describe "adding rules" do
 
-    let(:voyager_number) {
-      '869926275400212'
-    }
-
-    it "should validate number as voyager" do
-      CreditCardValidations::Detector.add_brand(:voyager, {length: 15, prefixes: '86'})
-      detector(voyager_number).valid?(:voyager).must_equal true
-      detector(voyager_number).voyager?.must_equal true
-      detector(voyager_number).brand.must_equal :voyager
-    end
-
-
-    describe "Add voyager rule" do
-      before do
-        CreditCardValidations::Detector.add_brand(:voyager, {length: 15, prefixes: '86'})
-      end
+      let(:voyager_number) { '869926275400212' }
 
       it "should validate number as voyager" do
-
+        CreditCardValidations::Detector.add_brand(:voyager, length: 15, prefixes: '86')
         detector(voyager_number).valid?(:voyager).must_equal true
         detector(voyager_number).voyager?.must_equal true
         detector(voyager_number).brand.must_equal :voyager
-
       end
 
-      describe "Remove voyager rule" do
+      describe "Add voyager rule" do
         before do
-          CreditCardValidations::Detector.delete_brand(:voyager)
+          CreditCardValidations::Detector.add_brand(:voyager, length: 15, prefixes: '86')
         end
 
-        it "should not validate number as voyager" do
-          detector(voyager_number).respond_to?(:voyager?).must_equal false
-          detector(voyager_number).brand.must_be_nil
+        it "should validate number as voyager" do
+          detector(voyager_number).valid?(:voyager).must_equal true
+          detector(voyager_number).voyager?.must_equal true
+          detector(voyager_number).brand.must_equal :voyager
         end
-      end
 
-    end
+        describe "Remove voyager rule" do
+          before do
+            CreditCardValidations::Detector.delete_brand(:voyager)
+          end
 
+          it "should not validate number as voyager" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.